src/spacepackets/ccsds/tm_frame.py

```diff
@@ -21,6 +21,17 @@ class TransferFrameDataFieldStatus:
     segment_len_id: int
     first_header_pointer: int
 
+    @classmethod
```

Comment:
classmethod is still a special type of static method.

I mean just a regular membe method:
```py
def pack(self):
    ....
```